docbook/reference/en/en-US/modules/Installation_Configuration.xml

```diff
@@ -306,12 +306,15 @@
         <title>Deploying a RESTEasy application to WildFly</title>
         <para>
             RESTEasy is bundled with WildFly and completely integrated as per the requirements of Jakarta EE.
-            You can use it with &ENTERPRISE-BEANS; and CDI and you can rely completely on WildFly to scan for and deploy your &REST-API; services and providers.
-            All you have to provide is your &REST-API; service and provider classes packaged
-            within a WAR either as POJOs, CDI beans, or &ENTERPRISE-BEANS;.
-            A simple way to configure an application is by simply providing an empty web.xml file.
-            You can of course deploy any custom servlet, filter
-            or security constraint you want to within your web.xml, but none of them are required:
+            A JAX-RS application can contain &ENTERPRISE-BEANS; and CDI.
+            WildFly scans the WAR file for the &REST-API; services and provider classes packaged in the
+            WAR either as POJOs, CDI beans, or &ENTERPRISE-BEANS;.
+        </para>
+        <para>
+            A simple way to configure an application is to provide a minimal web.xml file,
+            one that contains only the web-app element.  Of course custom servlets, filters
+            and security constraints among other configuration information can
+            be specified in the web.xml
         </para>

```

Comment:
I wonder if we should remove this as a `web.xml` is no longer required. Or maybe just remove the "minimal web.xml" part.